release/ray_release/job_manager/anyscale_job_manager.py

```diff
@@ -1,18 +1,17 @@
-import io
 import os
 import time
 import subprocess
 import tempfile
 from collections import deque
-from contextlib import redirect_stdout, redirect_stderr, contextmanager
+from contextlib import contextmanager
 from typing import Any, Dict, Optional, Tuple
 
 
 from anyscale.sdk.anyscale_client.models import (
     CreateProductionJob,
     HaJobStates,
 )
-from anyscale.controllers.job_controller import JobController, terminal_state
+from anyscale.controllers.job_controller import terminal_state
```

Comment:
`terminal_state` is basically a set of the keys of `job_status_to_return_code`.

so just write:

````
terminal_state = set(job_status_to_return_code.keys())
````

after the `job_status_to_return_code`.

and remove the import of the `job_controller` everywhere.

Reply:
nice!